app-shells/peco/peco-0.5.3.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
This requires a revbump :I 

I think you should leave the EAPI bump for next version release, and just change license in this one.

Reply:
This ebuild is not stable in any arch, are you sure that it requires a revbump?

Reply:
Yes, EAPI changes have a chance of affecting runtime behaviour and list of installed files. We always revbump on EAPI bump to be sure, it's a policy.

Reply:
Done.